### DISCUSSION TITLE
Hi i need help with lively wallpaper WEATHERFORECAST

### DISCUSSION THREAD

**Discussion by ConVolkWulf** [rocksdanister/lively#497] [Q&A]
hi i have a probem i am trying to do live weatherforecast on my lively wallpaper but idk its just not working for me i tried all the steps i searched on internet aswell how to do it but idk  i generated my api key aswell and i tried to put it in then i tried this: api.openweathermap.org/data/2.5/weather?q=London&appid=xxx i tried this command cause it was an example so i just put there my api key but didnt work idk man please could you help me with that. please someone help me.

**Comment by rocksdanister** [comment]
duplicate: https://github.com/rocksdanister/lively/issues/130
Also don't share your apikey with others, delete that and create a new one.

**Reply by ConVolkWulf** [reply]
yo but how do i set it up bro??????

**Reply by rocksdanister** [reply]
Right click wallpaper in library -> Customise -> Enter the key & city.
Hit Refresh, api key will take some time to activate so if you get error try after a day or so.

**Reply by ConVolkWulf** [reply]
the #130 has the same question and you gave him the link and there are 5 steps but i already told you i did them and it was not working... could you give me example please or show me or tell me how to do it please???

**Reply by ConVolkWulf** [reply]
its not working tho
i put there my api key and location and wasnt working please give me an example

**Reply by rocksdanister** [reply]
Please read all the instructions:
Hit Refresh, api key will take some time to activate so if you get error try after a day or so.
